### ISSUE TITLE
Add test coverage for covered counties

### ISSUE THREAD

**Issue by jessex** [Recidiviz/recidiviz-data#155]
us_pa_greene is the first scraper to descend from generic_scraper and was built in #147. We have previously added test coverage for us_ny and us_vt in earlier PRs, but those test structures have some design issues to consider. Beyond being extremely verbose and time-consuming, they don't actually do anything to ensure that the scraper works against the actual production website: they engineer downloads static html or json fixtures and then writes tests against those. So those unit tests are valuable in that they check the business logic against what we expect to be returned by requests to the website, but those fixtures could grow stale, and they may not be worth the time and effort required to setup at present.

tl;dr:
* Consider more dynamic forms of testing (this is a larger design task to be prioritzed)
* Consider cheaper ways to unit test the business logic in a child scraper
* Combine some combination of these to write unit tests for us_pa_greene

**Comment by ohinds** [comment]
I feel good about the design of BaseScraperTest, and that, combined with #346, makes this issue moot, I think.
